Examples/ObjectiveC/LiveDataExample.m

```diff
@@ -0,0 +1,50 @@
+
+#import "LiveDataExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLiveData = @"LiveDataExample";
+
+@interface LiveDataExample () <MGLMapViewDelegate>
+
+@property (nonatomic, strong, nullable) NSTimer *timer;
+@property MGLShapeSource *source;
+@end
+
+@implementation LiveDataExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle darkStyleURL]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    mapView.delegate = self;
+    
+    mapView.tintColor = [UIColor darkGrayColor];
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    // Add a source to the map. https://wanderdrone.appspot.com/ generates coordinates for simulated paths.
+    NSURL *url = [NSURL URLWithString:@"https://wanderdrone.appspot.com/"];
+    _source = [[MGLShapeSource alloc] initWithIdentifier:@"wanderdrone-source" URL:url options:nil];
+    [style addSource:_source];
+    
+    // Add an icon to the map to represent the drone's coordinate.
+    MGLSymbolStyleLayer *droneLayer = [[MGLSymbolStyleLayer alloc] initWithIdentifier:@"wanderdrone-layer" source:_source];
+    droneLayer.iconImageName = [MGLStyleValue valueWithRawValue:@"rocket-15"];
```

Comment:
I would clarify where this symbol is coming from, since its confusing that we didn't include the asset for it in this example. In this case, the SDK can automatically reference Maki icons listed here: https://www.mapbox.com/maki-icons/

Reply:
> In this case, the SDK can automatically reference Maki icons

But only because the initial style includes the Maki icons in its sprite sheet, right?